Gemfile

```diff
@@ -36,7 +36,12 @@ gem 'rspec_junit_formatter', '>= 0.5.1'
 # Merging branch coverage results does not work for old, unsupported rubies and JRuby
 # We have a fix up for review, https://github.com/simplecov-ruby/simplecov/pull/972,
 # but given it only affects unsupported version of Ruby, it might not get merged.
-gem 'simplecov', git: 'https://github.com/DataDog/simplecov', ref: '3bb6b7ee58bf4b1954ca205f50dd44d6f41c57db'
+if RUBY_VERSION.start_with?('3.4.')
```

Comment:
Seems like `https://github.com/DataDog/simplecov` has been archived, and Ruby 3.4 requires a newer existing version of `simplecov`. Should more versions than just 3.4 be switched to `gem 'simplecov'`?

Reply:
`simplecov` is used in CircleCI to collect coverage data in CircleCI. This fork contains the code we need to report all the ruby versions. Meaning this change need to be validated in CircleCI changes in: https://github.com/DataDog/dd-trace-rb/pull/4040

The source code we have been using has not changed for a long time. Perhaps its a good idea to follow the path of `datadog-ruby_core_source` to release its own gem. This can also avoid smashing Github very hard on this.

Reply:
Looking at https://github.com/simplecov-ruby/simplecov/pull/972/files, our change is a one-liner. I think we could try applying this change as a monkey-patch when we load rspec perhaps?

(I'd advise against getting too fork-happy, because the more things we fork, the more things we need to maintain, and we're already struggling to maintain one gem >_>)

Reply:
Monkeypatching (actually, vendoring that file, modifying it to our end) seems like the way to go here, plus it would remove that pesky git repo reference.

Reply:
After meeting with Oleg, I realized the existing `simplecov` specification works. No change is needed. Thanks for your help!